pytest/tests/sanity/state_sync1.py

```diff
@@ -0,0 +1,63 @@
+# Spins up one validating node. Wait until they reach height 40.
+# Start the second validating node and check that the second node can sync up before
+# the end of epoch and produce blocks and chunks.
+
+import sys, time
+
+sys.path.append('lib')
+
+
+from cluster import start_cluster
+
+BLOCK_WAIT = 40
+
+consensus_config = {"consensus": {"block_fetch_horizon": 10, "block_header_fetch_horizon": 10}}
+nodes = start_cluster(2, 0, 1, {'local': True, 'near_root': '../target/debug/'}, [["epoch_length", 80], ["block_producer_kickout_threshold", 10], ["chunk_producer_kickout_threshold", 10]], {0: consensus_config, 1: consensus_config})
+time.sleep(2)
+nodes[0].kill()
+
+cur_height = 0
+print("step 1")
+while cur_height < BLOCK_WAIT:
+    status = nodes[1].get_status()
+    cur_height = status['sync_info']['latest_block_height']
+    time.sleep(2)
+nodes[0].start(nodes[0].node_key.pk, nodes[0].addr())
```

Comment:
I'm confused that there is node 1 is working and node 0 is stopped, and then node 0 tries to get node 1. Personally, I prefer to keep the natural logic: 0 is the origin, 1 comes after 0, 2 comes after 1, and so on.
It's subjective.

Reply:
Ah good point. Will change that